src/cellvoyager_types/_metadata.py

```diff
@@ -111,12 +113,17 @@ class TargetWell(Base):
 
 
 class WellSequence(Base):
-    is_selected: bool
-    target_well: list[TargetWell]
+    is_selected: bool = Field(alias='IsSelected')
```

Comment:
We don't need to define aliases everywhere, thanks to:

https://github.com/fmi-faim/cellvoyager-types/blob/50efe7b99753110a78a1524a20165e24130bd514/src/cellvoyager_types/_metadata.py#L8